pcsx2/gui/Debugger/DebuggerLists.cpp

```diff
@@ -256,7 +256,8 @@ wxString BreakpointList::getColumnText(int item, int col) const
 				dest.Write(L"-");
 			} else {
 				char temp[256];
-				disasm->getOpcodeText(displayedBreakPoints_[index].addr, temp);
+				if(cpu->isAlive())
+					disasm->getOpcodeText(displayedBreakPoints_[index].addr, temp);
 				dest.Write("%s",temp);
```

Comment:
Haven't tested this yet, but if the cpu isn't alive, `temp` is uninitialised when you write its contents to `dest`.

Same applies to the other change.